Gemfile

```diff
@@ -6,6 +6,8 @@ gem 'ruby-fifo', github: 'shurizzle/ruby-fifo'
 
 group :development do
   gem 'rspec'
+  gem 'coveralls'
+  gem 'simplecov'  
```

Comment:
I did not realize that Gemfile and .gemspec served the same purpose. I will remove them from Gemfile. 

I use rubocop for stylechecker/linter. rspec handles the expectations (asserts) I use in cucumber. I will move those to gemspec as well.